client/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
Great to see that node_modules is inside the .gitignore file. It's important that we abstract node_modules from Version Control.